src/SPU.cpp

```diff
@@ -789,12 +799,28 @@ void Mix(u32 dummy)
     rightoutput = ((s64)rightoutput * MasterVolume) >> 7;
 
     leftoutput >>= 8;
+    rightoutput >>= 8;
+
+    // Add SOUNDBIAS value
+    // The value used by all commercial games is 0x200, so we subtract that so it won't offset the final sound output.
+    if (ApplyBias == true)
+    {
+        leftoutput += (Bias << 6) - 0x8000;
+        rightoutput += (Bias << 6) - 0x8000;
+    }
+
     if      (leftoutput < -0x8000) leftoutput = -0x8000;
     else if (leftoutput > 0x7FFF)  leftoutput = 0x7FFF;
-    rightoutput >>= 8;
     if      (rightoutput < -0x8000) rightoutput = -0x8000;
     else if (rightoutput > 0x7FFF)  rightoutput = 0x7FFF;
 
+    // The original DS and DS lite degrade the output from 16 to 10 bit before output
+    if (Degrade10Bit)
+    {
+        leftoutput &= 0xFFFFFFC0;
+        rightoutput &= 0xFFFFFFC0;
+    }
+
     // OutputBufferFrame can never get full because it's
     // transfered to OutputBuffer at the end of the frame
     OutputBackbuffer[OutputBackbufferWritePosition    ] = leftoutput >> 1;
```

Comment:
Extra whitespace here.
```suggestion
    OutputBackbuffer[OutputBackbufferWritePosition] = leftoutput >> 1;
```

Reply:
I didn't even touch that line, Arisotura did it like that.

Reply:
well, wouldn't hurt to fix it :D

Reply:
just realised that's to make it look nice.